core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/service/ReactorFlowManager.java

```diff
@@ -112,6 +113,13 @@ public void triggerStackRemoveInstance(Long stackId, String hostGroup, Long priv
         notify(selector, event);
     }
 
+    public void triggerStackRemoveInstances(Long stackId, String hostGroup, Set<String> hostnames) {
+        String selector = FlowChainTriggers.FULL_DOWNSCALE_TRIGGER_EVENT;
+        ClusterAndStackDownscaleTriggerEvent event = new ClusterAndStackDownscaleTriggerEvent(selector, stackId, hostGroup, hostnames,
```

Comment:
There is no constructor for setting the hostnames, only the CB private id is supported for now. This causes a compile error.